optional_plugins/html/avocado_result_html/__init__.py

```diff
@@ -309,9 +304,8 @@ def configure(self, parser):
 
         settings.add_argparser_to_option(
             namespace='job.run.result.html.enabled',
-            choices=('on', 'off'),
             parser=run_subcommand_parser,
-            long_arg='--html-job-result')
+            long_arg='--disable-html-job-result')
```

Comment:
I find this one mentioned in [documentation](https://avocado-framework.readthedocs.io/en/83.0/plugins/optional/results.html?highlight=%E2%80%93html-job-result#html-results-plugin) can you please also update  the document.

Reply:
Nice catch @richtja.